examples/mlflow_artifacts/docker-compose.yml

```diff
@@ -13,8 +16,8 @@ services:
       MINIO_ROOT_PASSWORD: "password"
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
-      interval: 1s
-      timeout: 10s
+      interval: 10s
+      timeout: 60s
```

Comment:
Can we file a different PR for this because it's unrelated to psutil installation?

Reply:
yep I'll remove this change.

Reply:
We should separate changes if separatable. Filing more PRs is better than making it difficult to revert or cherry-pick changes.